remotespark/livyclientlib/livysession.py

```diff
@@ -90,11 +89,14 @@ def create_sql_context(self):
         self.ipython_display.writeln("Creating HiveContext as 'sqlContext'")
         command = self._get_sql_context_creation_command()
         try:
-            command.execute(self)
+            (success, out) = command.execute(self)
         except LivyClientTimeoutError:
             raise LivyClientTimeoutError("Failed to create the SqlContext in time. Timed out after {} seconds."
                                          .format(self._wait_for_idle_timeout_seconds))
-        self.created_sql_context = True
+        if success:
```

Comment:
Start method - in LivySession Class - throw an exception if the session is not created (Id < 0), I can't find the relevant unit test for that? Can you point me to it to do the same?